opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraph.jsx

```diff
@@ -423,6 +416,16 @@ const groupingKnowledgeGraphStixRelationshipQuery = graphql`
   }
 `;
 
+const groupingKnowledgeGraphComponentStartInvestigationQuery = graphql`
+  query GroupingKnowledgeGraphComponenentStartInvestigationQuery($id: String!) {
+    grouping(id: $id) {
```

Comment:
If this feature is for all containers, we can have
```
container(id: $id) {
    startInvestigation {
        id
      }
}
```

And extract the logic

Reply:
This feature is for reports, groupings, incident responses rfis and rfts. Are they all the containers?

Reply:
Almost 😥
https://www.figma.com/file/Srp4IQ9xAnzaS043epUZuJ/OpenCTI---Models?type=whiteboard&node-id=0-1&t=1c7FborGg4IP1jWL-0

Reply:
Buuuut, you may have put the finger on something. It may worth a discussion with JP to determine if a new (sub)type is emerging. What are common to **rfts**, **rfis**, **incident responses**, **grouping**, **reports** (other than places to start investigations from) that are not to **Note**, **Opinions** and **observed data**

Reply:
And thx for pointing me the Model board 🙏